spec/support/boot.js

```diff
@@ -74,11 +74,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   var filterSpecs = !!queryString.getParam('spec');
 
   // Overrides
-  env.stopOnSpecFailure(true);
```

Comment:
These need putting back as they don't work from the grunt plugin config even without the boot.js